app/models/move.rb

```diff
@@ -93,6 +93,13 @@ class Move < VersionedModel
 
   scope :not_cancelled, -> { where.not(status: MOVE_STATUS_CANCELLED) }
 
+  attr_accessor :version
+
+  # TODO: Temporary method to apply correct validation rules when creating v2 move
```

Comment:
This can be removed when we drop support for v1 - along with the `set_move_type` method below